share/spice/sentiment_analysis/document/sentiment_analysis_document.js

```diff
@@ -0,0 +1,77 @@
+var ddg_spice_sentiment_analysis = {
```

Comment:
So, *$.getJSON* and *item.set('property', value)*  should be used only if JSONP is not available by the API provider. Is this correct?

Since the API I'm using supports JSONP, I'm adding *&callback={{callback}}* to the request's URL and not setting *wrap_jsonp_callback* to 1. When I hit the second API with *$.getScript(..)* , I parse the results and add it to the object that will be passed as argument of the function *render()* in order to all the data to be rendered. Am I doing it properly?